### PR TITLE
Allow cyborg combat module in code red

### PR DESCRIPTION
**Pull request by Eearslya**
As mentioned in the Discord chat, small change to automatically allow the Crisis module in code red without admin intervention.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -246,7 +246,7 @@
 		return
 	var/list/modules = list()
 	modules.Add(robot_module_types)
-	if((crisis && security_level == SEC_LEVEL_RED) || crisis_override) //Leaving this in until it's balanced appropriately.
+	if(crisis || security_level == SEC_LEVEL_RED || crisis_override) //Leaving this in until it's balanced appropriately.
```
Comment (SpadesNeil) [inline code/modules/mob/living/silicon/robot/robot.dm]: This should have **// Vorestation Edit** somewhere.

Comment (Eearslya) [inline code/modules/mob/living/silicon/robot/robot.dm]: Revised and pushed.

**Comment by SpadesNeil** [comment]
Marking this as "requires discussion" because this _is_ woefully imbalanced.

**Comment by SilverTalismen** [comment]
I personally don't know whats in the crisis module? is it that medical one or different?

**Comment by SpadesNeil** [comment]
Combat module, actually. Not crisis module.

**Comment by SilverTalismen** [comment]
Aaah, I was going to say, why is the crisis module, which is one of medicals, not allowed to be used unless its red alert xD

**Comment by Cameron653** [comment]
Going to note down the combat module equipment:

Flash
Thermal Vision 
Laser Gun
Plasma Cutter
Shield Module
Speed Module
Laser-Cannon (When emagged)

I can personally see this being a problem, mainly due to the thermal vision. 
This could cause events that cause the station to be on code red (Hostile invasion) being cut short because a cyborg saw them with thermals and called it out while the hostile was kidnapping/eating someone in a hidden away area.
Everything else looks fine, but thermals could cause major problems during events.

**Comment by SpadesNeil** [comment]
I opt to remove thermals.

**Comment by Eearslya** [comment]
Okay, after stumbling through git for a little bit...

Removed the thermals, and reduced the speed of the mobility module.

**Comment by SpadesNeil** [comment]
I'm still hesitant to add this only because I'm concerned about the ramifications of it, but fuck it, let's see it in action. This is something that can only be tested live.

**Pull request merged**